src/algebra/order/lattice_group.lean

```diff
@@ -72,17 +72,28 @@ variables {α : Type u} [lattice α] [comm_group α]
 @[to_additive]
 lemma mul_sup [covariant_class α α (*) (≤)] (a b c : α) : c * (a ⊔ b) = (c * a) ⊔ (c * b) :=
 begin
-  refine le_antisymm _ (by simp),
+  refine le_antisymm _ _,
```

Comment:
Can you surround the two subgoals below in braces?

Reply:
Yes, I think that's better.